wiki/Glossary/en.md

```diff
@@ -777,48 +779,62 @@ Star has four meanings:
    - Examples include adjustments in timing, incorrect beat spacing in some areas, or creating additional difficulty versions.
    - Beatmaps receiving a star are denoted by a star graphic next to their thread's title.
    - A star may be removed by another mod if the beatmap is thought to be in need of more improvements.
-2. another term for "Difficulty Stars"
+2. Another term for "Star Rating".
    - The symbol that represents the difficulty of a beatmap.
-3. another term for "[Kudosu](/wiki/kudosu)"
+3. Another term for "[Kudosu](#kudosu)".
    - These are given to beatmap modders and can be used to raise a beatmap above others.
-4. another term for "[Star Priority](/wiki/star_priority)"
+4. Another term for "[Star Priority](#star-priority)".
    - These stars are given to [osu!supporters](/wiki/osu!support) and are used in [Feature Requests](https://osu.ppy.sh/community/forums/4).
 
 ### Star Priority
 
 Star priority has two meanings:
 
-1. Star priority is a method of helping Beatmap Nominators to find beatmaps worthy of bubbling and later ranking.
+1. A method of helping Beatmap Nominators to find beatmaps worthy of bubbling and later ranking.
    - The star priority of a beatmap is determined by how many mod posts that have gained kudosu in the beatmap's thread, the sum of kudosu stars the map set has received.
-   - A kudosu star can be given by any user to any map, though shooting a kudosu star comes at the price of one kudosu.
+   - A kudosu star can be given by any user to any beatmap, though shooting a kudosu star comes at the price of one kudosu.
    - The conversion rate was formerly three kudosu for one kudosu star that increased the Star Priority by two.
    - The higher the star priority, the higher the beatmap's thread appears in the Works in Progress or Pending subforum.
-   - At least 12 star priority is needed for a map to be bubbled or ranked.
+   - At least 12 star priority is needed for a beatmap to be bubbled or ranked.
 2. A method in determining whether a Feature Request is wanted by the community.
    - The higher the star priority, the faster it gets noticed.
    - Users that have supporter status active when voting will shoot a pink star (![Kudosu](/wiki/shared/Kudos.png)) that increases Star Priority by 2 each, whilst a user without supporter will shoot a blue star (![Star](/wiki/shared/icon/star.gif)) that is worth only 1 each. Users may be able to shoot multiple stars depending on how many votes they have left.
 
+### Star Rating
+
+The visual representation of a beatmap's [difficulty](/wiki/Difficulties) version. This rating is (theoretically) between 0 and 10 stars based off of an algorithm.
+
+osu! currently uses two different algorithms to calculate star rating:
```

Comment:
"uses two different algorithms" but then you say the second one isn't in use 🤔 this section doesn't seem very useful without any description of algorithms